src/Symfony/Component/Config/Resource/DirectoryResource.php

```diff
@@ -68,7 +68,10 @@ public function isFresh($timestamp)
             return false;
         }
 
-        $newestMTime = filemtime($this->resource);
+        if ($timestamp <= filemtime($this->resource)) {
```

Comment:
this needs to be turned to a `<`: the equals case should not return false, as in eg FileResource